Update azure-stack-rotate-secrets.md

Paxilein opened the pull request:
The CertSharePath has to be in "" to prevent error-message:
The term '<NetworkPathofCertShare>' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.

Also the CertPassword can't be transformed into securestring in that way
-> Example 3 at https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.security/convertto-securestring?view=powershell-6#examples

azurecla commented:
Hi __@v-cmans__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-cmans). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

Commit:
Update azure-stack-rotate-secrets.md

The CertSharePath has to be in "" to prevent error-message:
The term '<NetworkPathofCertShare>' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.

Also the CertPassword can't be transformed into securestring in that way
-> Example 3 at https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.security/convertto-securestring?view=powershell-6#examples

mattbriggs reviewed (approved):
#sign-off
